### PR TITLE
Repo cleanup

### PR DESCRIPTION
**Pull request by dmi-try**

### CODE REVIEW

File: setup-env.sh [hunk 1]
```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-for agent in `ls mcagents/`; do
-    echo "Linking agent $agent of mcollective.."
-    ln -sf `readlink -f mcagents/$agent` /usr/libexec/mcollective/mcollective/agent/$agent
-done
-ln -sfT `readlink -f ../puppet/nailytest` /etc/puppet/modules/nailytest
-ln -sf `readlink -f ../puppet/nailytest/examples/site.pp` /etc/puppet/manifests/site.pp
-ln -sf `readlink -f ../bootstrap/sync/usr/bin/net_probe.py` /usr/bin/net_probe.py
-uuidgen > /etc/bootif  # for net_probe plugin
```
Comment (mihgen) [inline setup-env.sh:9]: I used this script in the past to setup the env for development. Let's make sure no one uses it: @rustyrobot, @warpc ?

Comment (yoojinl) [inline setup-env.sh:9]: I didn't use this script.

**Comment by mihgen** [comment]
+1

**Comment by adanin** [comment]
/tmp/hudson4894800439752200712.sh: 11: /tmp/hudson4894800439752200712.sh: license_check: not found
Build step 'Execute shell' marked build as failure
Finished: FAILURE

**Comment by warpc** [comment]
retest this please

**Pull request merged**